rust-toolchain.toml

```diff
@@ -1,5 +1,5 @@
 [toolchain]
-channel = "1.81.0"
+channel = "stable"
```

Comment:
We like to keep this pinned for consistency, unless your PR uses features on stable that aren't available in 1.81